codeworld-server/src/Build.hs

```diff
@@ -52,14 +55,19 @@ compileExistingSource mode programId = checkDangerousSource mode programId >>= \
         success <- runCompiler tmpdir userCompileMicros ghcjsArgs >>= \case
             Nothing -> return False
             Just output -> do
-                B.writeFile (buildRootDir mode </> resultFile programId) output
+                B.writeFile (buildRootDir mode </> rawResultFile programId) output
```

Comment:
Nothing Much, this is used just to store error message generated by ghcjs. I added this just to store the error message directly generated by ghcjs. If it is of not much use we remove this part.